components/server/src/projects/projects-service.ts

```diff
@@ -225,6 +225,18 @@ export class ProjectsService {
         return result;
     }
 
+    async getPrebuild(prebuildId: string): Promise<PrebuildWithStatus | undefined> {
```

Comment:
This should not reside in ProjectService I feel. It has no connection to that, at all. I'd rather see it in GitpodServerImpl - or a PrebuildsService, if we were to introduce one.

Reply:
My thought was because prebuilds are intended to only be used for projects, therefore I kept it here. But I can see why it seems out of place. I'll move it 👍🏼

Reply:
> I'll move it

:pray: